build.sbt

```diff
@@ -33,6 +34,20 @@ lazy val core = project
     )
   )
 
+lazy val formats = project
+  .in(file("formats"))
+  .dependsOn(core)
+  .settings(
+    name := "formats",
+    libraryDependencies ++= Seq(
+      "com.github.tototoshi" %% "scala-csv" % "1.3.10",
+      "org.scala-lang.modules" %% "scala-xml" % "2.1.0",
+      "io.spray" %% "spray-json" % "1.3.6",
```

Comment:
Should we consider using the `ujson` dependency in Joern to try to reduce the size of the class path?

Reply:
Of course, this would require re-implementing GraphSON

Reply:
That's a valid point, and I just gave it a try. Unfortunately the macro-generated json serializers produce different results and I didn't find a way to customise the auto-derivation. And creating manual `upickle.default.ReadWriter` for every involved class and trait amounts to a lot of boilerplate code. I've reverted those attempts for now, but if we want to pick it up later the beginnings are in https://github.com/joernio/flatgraph/pull/154/commits/200691c331fa73200a375ce02ae3e39d879b1d46 and https://github.com/joernio/flatgraph/pull/154/commits/ee41f0e70ecceaf27b1cef9737eaf496c32df3e3